src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceInt_getOffenderAlertsTest.java

```diff
@@ -13,7 +13,7 @@
 import static org.springframework.http.HttpMethod.GET;
 
 @DisplayName("BookingResource get offender alerts")
-public class BookingResourceIntTest_getOffenderAlerts extends ResourceTest {
+public class BookingResourceInt_getOffenderAlertsTest extends ResourceTest {
```

Comment:
test class name changed to have 'Test' at the end as otherwise it was being ignored